Documentation/dev/libvirt-howto.md

```diff
@@ -43,7 +43,19 @@ qemu-img resize coreos_production_qemu_image.img +8G
 #### 1.4 Get a Tectonic License
 Go to https://account.coreos.com/ and obtain a Tectonic license. Save the *pull secret* and *license path* somewhere.
 
-#### 1.5 Prepare the configuration file
+#### 1.5 Make sure you have permisions for qemu:///system
```

Comment:
nit: I'd prefer backtics for `qemu:///system`, but that's obviously not a big deal either way ;).